src/theme/typography.js

```diff
@@ -4,49 +4,59 @@ export default {
 
   h1: {
     fontWeight: 700,
-    fontSize: 35,
+    fontSize: 50,
     letterSpacing: "-0.24px",
```

Comment:
The sizes are too big according to me. 
This is the the typography you can look to https://github.com/codeforcauseorg/codeforcause.org/blob/HEAD/src/theme/typography.js